installer/scripts/service_control

```diff
@@ -143,6 +152,8 @@ restart_omsagent()
         return
     fi
 
+    warn_if_agent_not_onboarded
```

Comment:
Why not put this up at line 149? And: abort code flow if not onboarded, an error will just result.